docs/site/Relation-generator.md

```diff
@@ -0,0 +1,110 @@
+---
+lang: en
+title: 'Relation generator'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Relation-generator.html
+summary: summary
```

Comment:
Can you modify this to a summary of the generator or remove this line?

Reply:
Removed